.github/workflows/ci.yml

```
@@ -140,3 +140,31 @@ jobs:
         run: poetry run molecule test --scenario-name vars-priority
         env:
           TAILSCALE_CI_KEY: "${{ secrets.TAILSCALE_CI_KEY }}"
+
+  molecule-args:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

Needs this given new testing setup in this repo

```suggestion
      - uses: actions/checkout@v3
        with:
          ref: ${{ github.event.pull_request.head.sha }}
```